### PR TITLE
fix(Communication): do not create contacts if unchecked

### PR DESCRIPTION
**Pull request by vishdha**
TASK ID: [ASANA](https://app.asana.com/0/1198992588740629/1198992588764632)
Changes: 
[fix: do not create contacts](https://github.com/frappe/frappe/pull/10257)

### CODE REVIEW

File: frappe/email/doctype/email_account/email_account.json [hunk 1]
```diff
@@ -399,7 +399,7 @@
   }
  ],
  "icon": "fa fa-inbox",
- "modified": "2020-05-20 15:35:53.851057",
+ "modified": "2020-05-11 15:28:00.529049",
```
Comment (hrwX) [inline frappe/email/doctype/email_account/email_account.json]: - Where are the changes ?

**Review by hrwX — CHANGES_REQUESTED** [review]

File: frappe/email/doctype/email_account/email_account.json (hunk 1 above)
Comment (vishdha) [reply frappe/email/doctype/email_account/email_account.json]: https://github.com/frappe/frappe/pull/10257/commits/c3efc5eb018851cc86b97570873f4a4fe8b1d5ce

**Comment by hrwX** [comment]
@vishdha 
image

**Comment by vishdha** [comment]
close this PR all changes already available

**Comment by vishdha** [comment]
We already sync with existing changes hence closing this one

**Pull request closed without merging**